controllers/shipwrightbuild_controller.go

```diff
@@ -79,6 +81,19 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		logger.Error(err, "Retrieving ShipwrightBuild object from cache")
 		return RequeueOnError(err)
 	}
+	init := b.Status.Conditions == nil
+	if init {
+		b.Status.Conditions = make([]metav1.Condition, 0)
+		apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+			Type:    "ReconcileComplete",
+			Status:  metav1.ConditionUnknown, // we just started trying to reconcile
```

Comment:
Is there a better constant to reflect the fact that reconciliation is pending?

Reply:
done